### PR TITLE
Feat/st 564

### PR DESCRIPTION
**Pull request by muradalamv1**
###  https://tools.hmcts.net/jira/browse/ST-550
https://tools.hmcts.net/jira/browse/ST-554
https://tools.hmcts.net/jira/browse/ST-558
https://tools.hmcts.net/jira/browse/ST-564 ###

all the above tickets are included in this PR. these are very small changes, also the preview is there but no pdf will be created at this stage.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/resources/application.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ uk:
           welsh:
             SAVE_SIGN_OUT: '70dd0a1e-047f-4baa-993a-e722db17d8ac'
 
-idam:
+idam:lea
```
Comment (johnoneill46) [inline src/integrationTest/resources/application.yaml:32]: ?

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java [hunk 2]
```diff
@@ -28,6 +29,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         configBuilder.jurisdiction(JURISDICTION, "CIC", CcdServiceCode.CIC.getCcdServiceDescription());
 
 
+
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java:32]: Both of these seem a bit redundant

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/tab/CaseTypeTab.java [hunk 3]
```diff
@@ -219,5 +222,16 @@ private void buildCasePartiesTab(ConfigBuilder<CaseData, State, UserRole> config
...
 
+    }
+
+    private void buildOrderTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        configBuilder.tab("orders", "Orders")
+               .label("Orders",null,"### Orders")
+            .field("draftOrderTemplate")
+            .label("LabelState", null, "#### Case Status: ${[STATE]}")
+            .field("draftMainContentToBeEdited");
+
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/tab/CaseTypeTab.java:233]: remove redundant spacing

File: bin/idam-user-token.sh [hunk 4]
```diff
@@ -19,9 +19,9 @@ clientSecret=${OAUTH2_CLIENT_SECRET}
 redirectUri=http://localhost:3001/oauth2/callback
 
 if [ -z "$IDAM_STUB_LOCALHOST" ]; then
-  code=$(curl --insecure --fail --show-error --silent -X POST --user "${username}:${password}" "${IDAM_URL}/oauth2/authorize?redirect_uri=${redirectUri}&response_type=code&client_id=divorce" -d "" | docker run --rm --interactive stedolan/jq -r .code)
+  code=$(curl --insecure --fail --show-error --silent -X POST --user "${username}:${password}" "${IDAM_URL}/oauth2/authorize?redirect_uri=${redirectUri}&response_type=code&client_id=sptribs" -d "" | docker run --rm --interactive stedolan/jq -r .code)
 else
   code=stubbed-value
-fi
+if
```
Comment (johnoneill46) [inline bin/idam-user-token.sh:25]: Is 'fi' not the correct way to end a conditional in shell scripts?

**Review by johnoneill46 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java [hunk 5]
```diff
@@ -67,19 +71,34 @@ private void editDraftOrder(PageBuilder pageBuilder) {
...
             .build();
 
     }
 
     public SubmittedCallbackResponse draftCreated(CaseDetails<CaseData, State> details,
                                                   CaseDetails<CaseData, State> beforeDetails) {
         return SubmittedCallbackResponse.builder()
-            .confirmationHeader("Draft order edited")
+            .confirmationHeader(format("When you have finished drafting this order,you can send it to parties in this case."))
             .build();
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java:102]: Doesn't String.format expect some specifiers in the String? and if they aren't there (It's just a literal), you don't need to use format?

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java [hunk 6]
```diff
@@ -239,8 +239,10 @@ public class CaseData {
         label = "Select order template ",
         access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
     )
+    // private List<ListValue<DraftOrderCIC> draftOrderCIC = new ArrayList<>();
     private DraftOrderCIC draftOrderCIC = new DraftOrderCIC();
 
+
     @JsonUnwrapped(prefix = "link")
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java:246]: Redundant changes? Don't comment it out, remove it?

File: src/main/resources/application.yaml [hunk 7]
```diff
@@ -222,6 +222,7 @@ idam:
     microservice: sptribs_case_api
   api:
     url: ${IDAM_API_BASEURL:http://localhost:5000}
+
   systemupdate:
```
Comment (osheaf01) [inline src/main/resources/application.yaml:226]: Redundant space?

**Review by osheaf01 — COMMENTED** [review]
Not much to add to John's comments, please resubmit with changes and I'll re-review.

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java (hunk 2 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java:32]: These 2 lines of code has to be there until we resolved all the issues related to our Jurisdiction (ST_CIC). Once we get everything working then we will remove these 2 lines. that is what i think.

File: src/integrationTest/resources/application.yaml (hunk 1 above)
Comment (muradalamv1) [reply src/integrationTest/resources/application.yaml:32]: No Idea . cannot find the idam:iea   in code , and the Welsh has come from nfdiv. i believe we dnt need it so will be removed once we make sure that we dnt need Welsh.

**Review by ElifDenizEsea — APPROVED** [review]

File: src/integrationTest/resources/application.yaml (hunk 1 above)
Comment (johnoneill46) [reply src/integrationTest/resources/application.yaml:32]: Nothing to do with Welsh - looks like you've accidentally added "lea" onto the idam section

Comment (muradalamv1) [reply src/integrationTest/resources/application.yaml:32]: yes . it was there , now removed it

Comment (johnoneill46) [reply src/integrationTest/resources/application.yaml:32]: Have you pushed the changes? All looks to still be there

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java (hunk 6 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java:246]: when you remove the space then build fails.

File: src/integrationTest/resources/application.yaml (hunk 1 above)
Comment (muradalamv1) [reply src/integrationTest/resources/application.yaml]: yes. you can see changes now

**Review by sjamihmcts — APPROVED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

**Review by geciciserkan — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerDraftOrder.java [hunk 8]
```diff
@@ -71,7 +72,7 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
     public SubmittedCallbackResponse draftCreated(CaseDetails<CaseData, State> details,
                                                   CaseDetails<CaseData, State> beforeDetails) {
         return SubmittedCallbackResponse.builder()
-            .confirmationHeader("Draft order created")
+            .confirmationHeader(format("Draft order created"))
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerDraftOrder.java:75]: Is this format needed?

Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerDraftOrder.java:75]: no this does not need. this has been removed

**Review by sjamihmcts — DISMISSED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerDraftOrder.java [hunk 9]
```diff
@@ -14,6 +14,7 @@
 import uk.gov.hmcts.sptribs.common.ccd.PageBuilder;
 import uk.gov.hmcts.sptribs.document.content.DocmosisTemplateProvider;
 
+import static java.lang.String.format;
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerDraftOrder.java:17]: redundant import?

File: charts/sptribs-case-api/values.preview.template.yaml [hunk 10]
```diff
@@ -275,4 +275,6 @@ idam-pr:
       - https://xui-${SERVICE_FQDN}/oauth2/callback
     XUIMO:
       - https://xui-mo-${SERVICE_FQDN}/oauth2/callback
-
+    divorce:
```
Comment (johnoneill46) [inline charts/sptribs-case-api/values.preview.template.yaml:278]: What's the point of these?

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java [hunk 11]
```diff
@@ -13,18 +13,19 @@
 import uk.gov.hmcts.sptribs.ciccase.model.UserRole;
 import uk.gov.hmcts.sptribs.common.ccd.PageBuilder;
 
+import static java.lang.String.format;
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java:16]: redundant import?

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java [hunk 12]
```diff
@@ -12,6 +12,7 @@
 import uk.gov.hmcts.sptribs.common.ccd.CcdServiceCode;
 
 
+
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java:15]: remove redundant new spaces - no point modifying this file for no reason

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java (hunk 2 above)
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java:32]: same as above

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java [hunk 13]
```diff
@@ -241,6 +241,7 @@ public class CaseData {
     )
     private DraftOrderCIC draftOrderCIC = new DraftOrderCIC();
 
+
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java:244]: remove redundant new line

File: src/main/resources/application.yaml [hunk 14]
```diff
@@ -222,6 +222,7 @@ idam:
     microservice: sptribs_case_api
   api:
     url: ${IDAM_API_BASEURL:http://localhost:5000}
+
```
Comment (johnoneill46) [inline src/main/resources/application.yaml:225]: remove this line

**Review by johnoneill46 — CHANGES_REQUESTED** [review]

**Review by johnoneill46 — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Review by johnoneill46 — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**